__tests__/app.test.js

```diff
@@ -40,7 +40,29 @@ describe("/api/topics",()=>{
     })
 })
 
-describe("/api/topics",()=>{
+describe("/api/articles",()=>{
+    test("GET 200 and all articles upon request", () => {
+        return request(app)
+            .get("/api/articles")
+            .expect(200)
+                .then(({body})=>{
+                    const {articles} = body
+                    expect(articles.length).toBe(13)
+                    articles.forEach((article)=>{
+                        expect(typeof article.article_id).toBe("number")
```

Comment:
have a look at .toMatchObject and .objectContaining here for a better way to assert against objects, this is a bit hard to read